docs/how_to_contribute/math/how_to_add_prime_fields.md

```diff
@@ -10,6 +10,12 @@ Begin by creating a directory named `<new_prime_field>_prime` in `/tachyon/math/
 
 ### Prime field generator
 
+Tachyon currently supports 3 kinds of prime fields.
+
+- `Binary Prime Field`: This prime field is used in the STARK protocol, particularly in [Binius](https://eprint.iacr.org/2023/1784). It is generated when the modulus bits are one of [2¹, 2², 2⁴, 2⁸, 2¹⁶, 2³², 2⁶⁴, 2¹²⁸], The config is generated based on [binary_prime_field_config.h.tpl](/tachyon/math/finite_fields/generator/prime_field_generator/binary_prime_field_config.h.tpl), [binary_prime_field64_config.h.tpl](/tachyon/math/finite_fields/generator/prime_field_generator/binary_prime_field64_config.h.tpl) and [binary_prime_field128_config.h.tpl](/tachyon/math/finite_fields/generator/prime_field_generator/binary_prime_field128_config.h.tpl). The prime field for both CPU and GPU is instantiated by [binary_prime_field.h](/tachyon/math/finite_fields/binary_prime_field.h).
```

Comment:
```suggestion
- `Binary Field`: This prime field is used in the STARK protocol, particularly in [Binius](https://eprint.iacr.org/2023/1784). It is generated when the modulus bits are one of [2¹, 2², 2⁴, 2⁸, 2¹⁶, 2³², 2⁶⁴, 2¹²⁸], The config is generated based on [binary_prime_field_config.h.tpl](/tachyon/math/finite_fields/generator/prime_field_generator/binary_prime_field_config.h.tpl), [binary_prime_field64_config.h.tpl](/tachyon/math/finite_fields/generator/prime_field_generator/binary_prime_field64_config.h.tpl) and [binary_prime_field128_config.h.tpl](/tachyon/math/finite_fields/generator/prime_field_generator/binary_prime_field128_config.h.tpl). The prime field for both CPU and GPU is instantiated by [binary_prime_field.h](/tachyon/math/finite_fields/binary_prime_field.h).
```

Reply:
As I told you before, `Binary Prime Field` sounds more natural to me since it inherits `PrimeFieldBase`.